docs/API.md

```diff
@@ -551,19 +551,19 @@ Example response:
 An object with the following properties:
 
 * `"domain"`: A *domain name*, required. The zone to test.
-* `"ipv6"`: A boolean, optional. (default `false`). Used to configure the test and enable IPv4 tests.
-* `"ipv4"`: A boolean, optional. (default `false`). Used to configure the test and enable IPv6 tests.
-* `"nameservers"`: A list of *name server* objects, optional. Used to perform un-delegated test.
-* `"ds_info"`: A list of *DS info* objects, optional. Used to perform un-delegated test.
+* `"ipv6"`: A boolean, optional. (default `true`). Used to configure the test and enable IPv4 tests.
+* `"ipv4"`: A boolean, optional. (default `true`). Used to configure the test and enable IPv6 tests.
+* `"nameservers"`: A list of *name server* objects, optional. (default: `[]`). Used to perform un-delegated test.
```

Comment:
In the current implementation, if "nameservers" is given, it can be an empty array. The final behavior will be the same as if the property was not given because the engine will ignore it.

So, if we are strict about our implementation, we must to say that there are no default value or if are not so strict and we want communicate about the global behavior, we keep this default value for "nameservers" and "ds_list".